### ISSUE TITLE
Global dice coefficient not improving after initial 50 epochs

### ISSUE THREAD

**Issue by karthikk2085** [MIC-DKFZ/nnUNet#604]
Hi Fabian,

After the initial 50 epochs, I don't see much improving in the validation global dice coefficient value on lung CT segmentation dataset. Currently it has run for around 400 epochs and I don't see any improvement huge on the validation data for each of the folds in 2d configuration. Could you let me know what effects could I expect in the training process if I restrict my training process to just 50 epochs?

**Comment by FabianIsensee** [comment]
Hi, please just let the training finish. The learning rate schedule is not designed to be terminated early
Best,
Fabian
